### PR TITLE
Dev version 1.5 Uber Map the Leaf

### PR DESCRIPTION
**Pull request by byarsawich**

### CODE REVIEW

File: src/aura/LeafletUberMapper/LeafletUberMapper.cmp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<aura:component controller="LeafletUberMapperController" implements="force:appHostable,flexipage:availableForAllPageTypes" access="global">
+S<aura:component controller="LeafletUberMapperController" implements="force:appHostable,flexipage:availableForAllPageTypes" access="global">
```
Comment (Fitchburgh) [inline src/aura/LeafletUberMapper/LeafletUberMapper.cmp]: What's the "S" for?

File: src/objects/Asset.object [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Fitchburgh) [inline src/objects/Asset.object]: Does this need to declared anywhere in the objectPackage file?

Comment (byarsawich) [reply src/objects/Asset.object]: we could add it to make it easier to copy over to a package file

Comment (byarsawich) [reply src/objects/Asset.object]: It's already in there

**Pull request merged**